data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -165,9 +165,18 @@ spec:
                       description: Azure is the configuration used when installing
                         on Azure.
                       properties:
+                        encryptionAtHost:
+                          description: EncryptionAtHost enables encryption at the
+                            VM host.
+                          type: boolean
                         osDisk:
                           description: OSDisk defines the storage for instance.
                           properties:
+                            encryptionType:
+                              description: encryptionType is the type of encryption used for disks.
+                                Possible values include: 'EncryptionAtRestWithPlatformKey',
```

Comment:
This type, as with the other types for disk encryption sets, is here: 

https://github.com/Azure/azure-sdk-for-go/blob/165ee2301805cbda1463088ae0dd403a14395a1f/services/compute/mgmt/2020-06-01/compute/enums.go#L376 . 

The encryption type is a property of DiskProperties: 

https://github.com/Azure/azure-sdk-for-go/blob/165ee2301805cbda1463088ae0dd403a14395a1f/services/compute/mgmt/2020-06-01/compute/models.go#L3390 

DiskProperties -> Encryption -> EncryptionType. 

I am not familiar with openshift/api, however it does not appear to map 1 to 1 to the Azure API, so I placed the encryptionType under osDisk since it was the closest mapping. 

After reading what you wrote above, I am in agreement that we should not provision the keyvault and keys.